lib/lm3s/rcc.c

```diff
@@ -0,0 +1,105 @@
+/** @defgroup rcc_file RCC Controller
+
+@brief <b>LM3S RCC Controller</b>
+
+@ingroup LM3Sxx
+
+@version 1.0.0
+
+@author @htmlonly &copy; @endhtmlonly 2011
+Daniele Lacamera <root at danielinux dot net>
+
+@date 21 November 2015
+
+LGPL License Terms @ref lgpl_license
+*/
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+ * Copyright (C) 2009 Federico Ruiz-Ugalde <memeruiz at gmail dot com>
+ * Copyright (C) 2011 Fergus Noble <fergusnoble@gmail.com>
+ * Copyright (C) 2011 Stephen Caudle <scaudle@doceme.com>
+ * Copyright (C) 2015 Daniele Lacamera <root@danielinux.net>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "stdint.h"
+#include "libopencm3/lm3s/rcc.h"
+
+extern void __dmb(void);
+
+
+/* PLL/Systick Configured values */
+#define QEMU_SYSDIV_VAL        (3)
+#define QEMU_PWMDIV_VAL        (7)
+#define QEMU_XTAL_VAL          (14)
```

Comment:
These feel like poor names.  I'm totally totally interested in being able to run libopencm3 code under qemu, but some magic qemu defines definitely seems out of scope.  Surely it should just be, "init to these values, which are _also_ totally normal useful values, not just for qemu, and it will work well with qemu"  I'm not really sure I see why qemu needs it's own defines here.